### DISCUSSION TITLE
Link your Slack to Matrix

### DISCUSSION THREAD

**Discussion by leethax666** [streetcomplete/StreetComplete#4862] [Ideas]
You should have a matrix channel, a lot of FOSS type people prefer it. I can help set it up. https://app.element.io/ is the most popular homeserver, but there are others

**Comment by matkoniecz** [comment]
EDITED

https://streetcomplete.app/ lists Slack but we do not administer or control it. Bridging US Slack should be discussed with whoever controls and moderates it.

Personally I prefer to not deal with moderating one more channel - I should spend less time on OSM discussions, not more. (and there are some extra issues there)

Also, I have not yet really figured out Element (it seems to be an official Matrix app for Android?). For example - how to stay subscribed to channel but to hide it from home page visible on opening the app.

For discussions/issues that should reach developers this repository and its discussion places are the best place.

----------------

ORIGINAL:

We do not have Slack, therefore there is no Slack to link to Matrix. I assume at least that bridging is done by people controlling Slack/Discord/etc instances.

(There may be dedicated StreetComplete channel at some Slack/Discord/etc. servers and on some people connected to StreetComplete may be active. But for example #streetcomplete at OSM US Slack is not moderated by any of us, bridging US Slack should be discussed with whoever controls it.)

For discussions/issues that should reach developers this repository and its discussion places are the best place.

**Reply by matkoniecz** [reply]
Unless US Slack (or other Slack) is mentioned somewhere here as our Slack?

**Comment by leethax666** [comment]
https://streetcomplete.app/ lists one

Sent with [Proton Mail](https://proton.me/) secure email.

------- Original Message -------

**Reply by matkoniecz** [reply]
thanks, I updated https://github.com/streetcomplete/StreetComplete/discussions/4862#discussioncomment-5223188

**Comment by westnordost** [comment]
The channel on slack US is kind of official, but I have no idea how to link that to matrix. As Mateusz wrote, it is just another channel on slack US, noone (as far as I know) is admin of that channel.

**Reply by matkoniecz** [reply]
I am definitely no an admin/moderator there or having any elevated user rights.

**Comment by leethax666** [comment]
if anyone is in the slack they should show the admins this https://matrix.org/docs/projects/bridge/matrix-appservice-slack

Sent with [Proton Mail](https://proton.me/) secure email.

------- Original Message -------
